### ISSUE TITLE
Report invalid runtimes

### ISSUE THREAD

**Issue by pritidesai** [apache/incubator-openwhisk-wskdeploy#307]
There are only few valid runtimes available:

```
Runtime name | Supported versions | Default | Description
nodejs | nodejs@6, nodejs@5 | nodejs@6 | Node.js runtime.
java | java@8 | java@8 | Java language runtime.
python | python@2.7 | python@2.7 | Python language runtime.
swift | swift@3, swift@2 | swift@3 | Swift language runtime.
language:default | N/A | N/A | Permit the OpenWhisk platform to select the correct default language runtime.
```

manifest_parser should report if a runtime is set to any value outside of the above valid options.

For example, today when I say my runtime is set to "invalid", ComposeActions accepts it as is and does not complain:

```
`package:
   name: helloworld
   actions:
     helloInvalidRuntime:
       location: ../tests/usecases/helloworld/actions/hello.js
       runtime: invalid`
```

**Comment by lzbj** [comment]
@pritidesai , is it related to #306 , I suggest we combine them.

**Comment by daisy-ycguo** [comment]
Fixed in PR #491. Close it as fixed.
